src/main/kotlin/no/nav/helse/oppslag/person/PersonClient.kt

```diff
@@ -12,20 +12,25 @@ class PersonClient(private val personV3: PersonV3) {
 
     fun personInfo(id: AktørId) =
             Try {
-                personV3.hentPerson(hentPersonRequest(id)).person
+                personV3.hentPerson(hentPersonRequest(id, listOf(Informasjonsbehov.ADRESSE))).person
```

Comment:
Trenger man adresse for å hente ut barn?

Reply:
tenkte det var logisk at "Person"-objektet er likt på person/{aktør} som listen man får på /person/{aktør}/barn ?

Reply:
Kanskje vi burde heller laget et `Barn`-objekt, som er lik `Person` bare uten `adresse`

Reply:
Brukes adressen til noe etterpå? Hvis ikke bør vi ikke hente den.

Reply:
Nei sant, kan lage et Barn-objekt som har et mindre datasett og slå opp de uten noen informasjonsbehov